assets/.prettierrc

```diff
@@ -1,15 +1,16 @@
 {
-  "printWidth": 1300,
```

Comment:
I need to see if this will have any problems when formatting through PhpStorm

Reply:
It's Prettier it works that way, it's so necessary for strandart. 1300 is too much, same as infinity 😩